src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -1,26 +1,31 @@
-import React, { ReactEventHandler, useEffect, useRef, useState } from 'react';
+import React, { Fragment, useEffect, useState } from 'react';
 import { OktaContext } from '@okta/okta-react';
 
 const storageKey = 'dev-user-config';
 
+const jobCodes = ['EASI_D_GOVTEAM', 'EASI_P_GOVTEAM'];
+type EuaJobCodes = typeof jobCodes[number];
+type EuaJobCodeMap = { [jobCode in EuaJobCodes]: boolean };
+
 const initialAuthState = {
   isAuthenticated: false,
-  isPending: false,
-  name: '',
-  euaId: '',
-  groups: [] as string[]
+  isPending: false
 };
 
-const jobCodes = ['EASI_D_GOVTEAM'];
-
 type ParentComponentProps = {
   children: React.ReactNode;
 };
 
 const DevSecurity = ({ children }: ParentComponentProps) => {
-  const [authState, setAuthState] = useState(initialAuthState);
+  const [authState, setAuthState] = useState<{
+    isAuthenticated: boolean;
+    isPending: boolean;
+  }>(initialAuthState);
   const [euaId, setEuaId] = useState('');
-  const checkboxValues = useRef(new Set<string>(jobCodes));
+  const [groups, setGroups] = useState<EuaJobCodeMap>({
```

Comment:
I decided to make the groups/job codes in an object with a boolean value to indicate whether the job code is enabled or not.

I made this change because I have regrets with using arrays for checkboxes with the system intake collaborators. I thought I'd give this a try and it works pretty well (typescript hiccups along the way).